crates/typst/src/model/numbering.rs

```diff
@@ -208,8 +208,11 @@ impl FromStr for NumberingPattern {
             };
 
             let prefix = pattern[handled..i].into();
-            let case =
-                if c.is_uppercase() || c == '壹' { Case::Upper } else { Case::Lower };
+            let case = if c.is_uppercase() || c == '壹' || c == 'Α' {
```

Comment:
why `c == 'Α'`? isn't `c.is_uppercase()` true for it?

probably we need to update `c.to_ascii_lowercase()` above since we have more than ASCII here

Reply:
> why `c == 'Α'`? isn't `c.is_uppercase()` true for it?

Oh that is true; we do not need to do this. Updated.

> probably we need to update `c.to_ascii_lowercase()` above since we have more than ASCII here

Changed to use general to_lowercase()